src/c++/perf_analyzer/genai-perf/genai_perf/wrapper.py

```diff
@@ -40,8 +40,8 @@ def add_protocol_args(args):
         cmd = ""
         if args.service_kind == "triton":
             cmd += f"-i grpc --streaming "
-            if "url" not in vars(args).keys():
-                cmd += f"-u {DEFAULT_GRPC_URL}"
+            if "u" not in vars(args).keys():
```

Comment:
doesn't it need to look for both url and u? Or does the args collapse both down to just u?

Reply:
apparently this is correct